crawl-ref/source/timed-effects.cc

```diff
@@ -744,6 +744,7 @@ void monster::timeout_enchantments(int levels)
         case ENCH_FRIENDLY_BRIBED: case ENCH_CORROSION: case ENCH_GOLD_LUST:
         case ENCH_RESISTANCE: case ENCH_HEXED: case ENCH_IDEALISED:
         case ENCH_BOUND_SOUL: case ENCH_STILL_WINDS: case ENCH_RING_OF_THUNDER:
+        case ENCH_RING_OF_FLAMES:
```

Comment:
Why are Ring of Thunder and Ring of Flames (both permanent enchantments anyhow) in this function? Neither should be here as far as I can tell...

Reply:
Right. They are both getting removed entirely